controllers/api/v1/videoAnalysis.js

```diff
@@ -0,0 +1,189 @@
+const VideoAnalysisModel = require('../../../models/videoAnalysis');
+const utils = require('../../../lib/utils');
+const Joi = require('@hapi/joi');
+const ObjectId = require('mongodb').ObjectId;
+
+
+function getVideoAnalysisList(req, res, next) {
+    const logger = req.logger;
+    const clientId = req.query.client;
+    utils.setLogTokens(logger, 'videoAnalysis', 'getVideoAnalysisList', req.query.client, null);
+    let model = new VideoAnalysisModel(logger);
+    return model.getVideoAnalysisList(
+        req.app.kraken,
+        clientId,
+        req.query.videoId,
+        req.query.sortBy,
+        req.query.sortAsc,
+        req.query.limit,
+        req.query.next,
+        req.query.previous
+    ).then((result) => {
+        if (result) {
+            res.status(200).json(result);
+            return;
+        }
+        res.sendStatus(404);
+    }).catch(next);
+}
+
+
+function getVideoAnalysisDetails(req, res, next) {
+    const logger = req.logger;
+    const clientId = req.query.client;
+    utils.setLogTokens(logger, 'videoAnalysis', 'getVideoAnalysisDetails', req.query.client, null);
+    let model = new VideoAnalysisModel(logger);
```

Comment:
create model object outside all functions and resuse it (change it to const instead of let)